src/fsharp/service/IncrementalBuild.fs

```diff
@@ -1390,6 +1392,39 @@ type IncrementalBuilder(tcGlobals,
                 // Never open PDB files for the language service, even if --standalone is specified
                 tcConfigB.openDebugInformationForLaterStaticLinking <- false
 
+                tcConfigB.xmlDocInfoLoader <-
+                    { new IXmlDocumentationInfoLoader with
+                        /// Try to load xml documentation associated with an assembly by the same file path with the extension ".xml".
+                        /// If that fails, look to see if it exists in ILModuleDef's resources.
+                        member _.TryLoad(assemblyFileName, ilModule) =
+                            let xmlFileName = Path.ChangeExtension(assemblyFileName, ".xml")
```

Comment:
We should first look for the localized XML doc file, see https://docs.microsoft.com/en-us/dotnet/core/install/localized-intellisense

Reply:
e.g. if FSharp.Core.dll is at 

    C:\Users\Administrator\.nuget\packages\fsharp.core\5.0.1\lib\netstandard2.0\FSharp.Core.dll

then if we actually shipped localized XML doc for FSHarp.Core then it would be located at

    C:\Users\Administrator\.nuget\packages\fsharp.core\5.0.1\lib\netstandard2.0\de\FSharp.Core.xml

    C:\Users\Administrator\.nuget\packages\fsharp.core\5.0.1\lib\netstandard2.0\zh-hans\FSharp.Core.xml

etc.  I don't know the exact rules for how to look, e.g. does it search  through preferred English language versions:

    C:\Users\Administrator\.nuget\packages\fsharp.core\5.0.1\lib\netstandard2.0\en-us\FSharp.Core.xml
    C:\Users\Administrator\.nuget\packages\fsharp.core\5.0.1\lib\netstandard2.0\en-gb\FSharp.Core.xml
    C:\Users\Administrator\.nuget\packages\fsharp.core\5.0.1\lib\netstandard2.0\en\FSharp.Core.xml
    C:\Users\Administrator\.nuget\packages\fsharp.core\5.0.1\lib\netstandard2.0\FSharp.Core.xml

Reply:
After talking with @brettfo and @KevinRansom - it seems .NET as a whole does not handle localizations for XML documentation on libraries.

We could, in theory, add support for it ourselves, but I would wait until .NET as a whole decides what the rules are, whenever or if that happens.